packages/shared/lib/core/profile-manager/actions/events-handlers/handleTransactionInclusionEvent.ts

```diff
@@ -61,15 +61,15 @@ function handleGovernanceTransactionInclusionEvent(
         closePopup(true)
 
         const account = get(activeAccounts)?.find((_account) => _account.index === accountIndex)
-        if (account.hasVotingPowerTransactionInProgress) {
+        if (account?.hasVotingPowerTransactionInProgress) {
```

Comment:
we dont want to go into `else` if  account would be undefined

```suggestion
        if (!account) {
             return
        }
        if (account.hasVotingPowerTransactionInProgress) {
```